### ISSUE TITLE
JSON file import help links to MM help page

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#6104]
## Description 

JSON file import help links to MM help page

## Steps to Reproduce

   1. Enable crypto wallet
   2. Go to profile and connect hardware
   3. Select `Import`, Select JSON from file type
   4. `File import not working? Click here!` links to [MM help page](https://metamask.zendesk.com/hc/en-us/articles/360015489351-Importing-Accounts) 

## Actual result:

JSON file import help links to MM help page

## Expected result:
Should link to Brave support page on help center

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 0.71.75 Chromium: 77.0.3865.75 (Official Build) dev (64-bit)
-- | --
Revision | 201e747d032611c5f2785cae06e894cf85be7f8a-refs/branch-heads/3865@{#776}
OS | Linux

## Version/Channel Information:

- Can you reproduce this issue with the current release? N/A
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome?  N/A

## Miscellaneous Information:

cc: @Brave-Matt @atsyed87  to create Brave specific page
@bbondy @tomlowenthal

**Comment by bbondy** [comment]
This one is not fixed yet so moving it to the next milestone. FYI @ryanml

**Comment by srirambv** [comment]
@ryanml this one doesn't seem to be fixed on component `0.1.37`. I still see MM link being used

Link on [line 25](https://github.com/brave/ethereum-remote-client/pull/111/files#diff-7569d1789a1e186dd5ab4952333abe8bR25) seems to be wrong. 
```js
  replace(
          /https:\/\/metamask\.zendesk\.com\/hc\/en-us\/articles\/360015489351-Importing-Accounts/gm,
          'https://support.brave.com/hc/en-us/articles/360035488071-How-do-I-manage-my-Crypto-Wallets-'
        )
```
This is the acutal article link https://metamask.zendesk.com/hc/en-us/articles/360015489331-Importing-an-Account that needs to be replaced

**Comment by srirambv** [comment]
Verification passed on 

Brave | 1.6.18 Chromium: 80.0.3987.87 (Official Build) nightly (64-bit)
-- | --
Revision | 449cb163497b70dbf98d389f54e38e85d4c59b43-refs/branch-heads/3987@{#801}
OS | macOS Version 10.15.1 (Build 19B88)
Component | `0.1.43`

- Verified JSON file import help links to [Brave support page](https://support.brave.com/hc/en-us/articles/360035488071-How-do-I-manage-my-Crypto-Wallets-)
--------
Verification passed on 

Brave | 1.3.115 Chromium: 80.0.3987.87 (Official Build) (64-bit)
-- | --
Revision | 449cb163497b70dbf98d389f54e38e85d4c59b43-refs/branch-heads/3987@{#801}
OS | Linux
Component | `0.1.43`

- Verified JSON file import help links to  [Brave support page](https://support.brave.com/hc/en-us/articles/360035488071-How-do-I-manage-my-Crypto-Wallets-)
---------
Verification passed on 

Brave | 1.3.91 Chromium: 79.0.3945.117 (Official Build) beta (64-bit)
-- | --
Revision | 04f0a055010adab4484f7497fbfdbf312c307f1d-refs/branch-heads/3945@{#1019}
OS | Windows 10 OS Version 1809 (Build 17763.914)
Component | `0.1.43`

- Verified JSON file import help links to [Brave support page](https://support.brave.com/hc/en-us/articles/360035488071-How-do-I-manage-my-Crypto-Wallets-)
